src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -457,7 +457,7 @@
   </div>
 </div>
 
-{% if paymentOption == 'By instalments' or paymentOption == 'By a set date' %}
+{% if (paymentOption == 'By instalments' or paymentOption == 'By a set date') and  defendant.partyType == "INDIVIDUAL" %}
```

Comment:
Are sole traders not dealt in same way? Does that needs to be added to if block to separated out to filter out companies/org?
I know we haven't written response PDF aat/functional test but I think we should write one now making sure PDF is generated with correct contents?

Reply:
Also how about checking `statementOfMeans is defined` rather than `partyType` ?